docs/debugging_tips.md

```diff
@@ -0,0 +1,54 @@
+# Debugging Tips
+
+## Stacktraces for Rollup Loop Panics
+
+1. Start geth in a new terminal: 
+   ```bash
+   make start-geth-locally
+   ```
+
+2. Set up your environment:
+   ```bash
+   export HUBBLE_ETHEREUM_CHAIN_ID=1337
+   export HUBBLE_ETHEREUM_RPC_URL=ws://localhost:8546
+   export HUBBLE_ETHEREUM_PRIVATE_KEY=ee79b5f6e221356af78cf4c36f4f7885a11b67dfcc81c34d80249947330c0f82
```

Comment:
I prefer to set config options in `commander-config.yaml` and `deployer-config.yaml`. No need to worry about terminal sessions this way.